packages/terra-graphs-docs/src/example-datasets/dataObjects/Gantt/actionDataset1.js

```diff
@@ -0,0 +1,18 @@
+import {
+  loadPopup,
```

Comment:
nit: remove the comma to make it one line

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/229/commits/d8e8b462e04dc4f262cd888740100453f7d02e4a#diff-335c169402cf4d2243fa75dcc03446cca975f43c3e8ea13edd916681ebb680a5R1).